tools/dav1d.rs

```diff
@@ -91,28 +92,38 @@ use std::ffi::c_uint;
 use std::ffi::c_ulonglong;
 use std::ffi::c_void;
 use std::ptr::NonNull;
+use std::time::Duration;
 
-unsafe fn get_time_nanos() -> u64 {
-    let mut ts: libc::timespec = libc::timespec {
-        tv_sec: 0,
-        tv_nsec: 0,
-    };
-    libc::clock_gettime(1, &mut ts);
-    return (1000000000 as c_ulonglong)
-        .wrapping_mul(ts.tv_sec as c_ulonglong)
-        .wrapping_add(ts.tv_nsec as c_ulonglong) as u64;
+cfg_if::cfg_if! {
```

Comment:
Why are we using `cfg_if` instead of just putting a `cfg` on the function itself?